templates/client-daemonset.yaml

```diff
@@ -217,13 +217,13 @@ spec:
                 -hcl='verify_server_hostname = true' \
                 {{- end }}
                 {{- end }}
-                -hcl='ports { https = 8501 }' \
+                -hcl='ports { https = {{ .Values.client.ports.https.port }} }' \
                 {{- if .Values.global.tls.httpsOnly }}
                 -hcl='ports { http = -1 }' \
```

Comment:
Needs to be extended by an else condition and add hcl parameter to adjust http port